### PR TITLE
Widgets Testing

### PR DESCRIPTION
**Pull request by Kevin-Guillen**
appbar
detailsimage
maininfo
overview

### CODE REVIEW

File: test/custom_appbar_test.dart [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  Widget _buildWidget() {
+    return MaterialApp(
+      home: Scaffold(
+        body: MovieDetailsImage(
+          movieImage:
+              'https://www.themoviedb.org/t/p/w1280/qa6HCwP4Z15l3hpsASz3auugEW6.jpg',
+        ),
+      ),
+    );
+  }
```
Comment (MarcosCGlobant) [inline test/custom_appbar_test.dart]: you could abstract this method to another file, and pass the widget to test via parameter

**Review by FranciscoGaraventa — APPROVED** [review]
Resolve Harry comment, but then LGTM

File: test/custom_appbar_test.dart (hunk 1 above)
Comment (Kevin-Guillen) [reply test/custom_appbar_test.dart]: Done , also corrected line 10 where i was passing the incorrect widget.

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**